agent/jvm/pom.xml

```diff
@@ -96,16 +96,15 @@
       <plugin>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.6</source>
-          <target>1.6</target>
+          <source>1.7</source>
+          <target>1.7</target>
```

Comment:
I guess this jump is required, and I'm not against it. But we need to bump the version number accordingly, to indicate that the release is not compatible with Java 1.6. anymore. Also, I'd prefer to have this as a global switch for all components (not only for the JVM agent)

Reply:
It is required, when I tried to run the tests using JDK17 (as it no longer can handle `target=1.6`), but it's not a requirement for any other change. I can skip this change.

Reply:
Maybe we can skip it for now, and consider to bump to 1.7 as minimum in a followup PR (and then already on the overall level) ?

Reply:
I reverted this change, so it's still 1.6. 1.7 was needed if I wanted to build using JDK17. But I needed it to _test_ using JDK17. After verifying it works, we can leave at 1.6 value ;)